### ISSUE TITLE
A few questions about usage

### ISSUE THREAD

**Issue by Elaws** [mtkennerly/ludusavi-playnite#44]
So I've just installed `ludusavi` and `ludusavi-playnite`, and made a backup in Playnite using extensions > ludusavi > backup save data for all games.

Now, a few questions : 

1. Where is the list of saved games and the date of the save ? Can we visualize it in Playnite or Ludusavi standalone ? Should we load the backup inside Ludusavi ?
2. In Ludusavi standalone, should we specify the same backup path as in Playnite ?
3. Is there a way to know to which game executable (path) corresponds each save ?
4. Can we check the status for each save (e.g. : never saved yet because new game, save is up to date, save needs backup, etc...) ?

Thanks for the clarification, and for this amazing software !

**Comment by mtkennerly** [comment]
Hi! I'm glad you've found Ludusavi helpful :D

1. After doing the backup, you can click on the notification in Playnite for a list of all games and their backup sizes. You can also right click on any game in Playnite and use the Ludusavi context menu to see a list of backups for it. (If Ludusavi and Playnite have different names for the game, it may not show the backups in the context menu directly.)
2. That's up to you. For example, if you have auto-backups enabled during gameplay in Playnite, you might want to keep those separate from your manual backups from using Ludusavi on its own. You certainly can set them to the same path if you prefer to combine them.
3. Could you elaborate on this question? What are you trying to do/accomplish specifically?
4. Right now, the best way is to open Ludusavi separately and do a preview. It will show the `+` and `Δ` icons next to each game. The Playnite plugin will show these statuses after performing a backup (via the notification in Playnite), but it doesn't let you preview them beforehand.
5. If you've configured any custom games, then you'll probably want to back up `%APPDATA%/ludusavi/config.yaml`. Ludusavi  will automatically redownload `manifest.yaml` and recreate `cache.yaml` if necessary, so you don't have to back those up, but you certainly can back them up as well.

**Comment by Elaws** [comment]
Thanks for taking time to answer !

> Could you elaborate on this question? What are you trying to do/accomplish specifically?

Sometimes, I have several versions of a game, that maybe do not have their save folders in the same location, and I would like to check to which game executable corresponds a save in Ludusavi. Is this possible (something like right-click > show game folder) ?

**Comment by mtkennerly** [comment]
Ah, I see. You can't automatically jump to the game folder, but you can do it with a few extra steps by checking the game's file list. For example, I have a few copies of Dead Cells, so I can do this:

> https://user-images.githubusercontent.com/4934192/205176520-60d4b6eb-e0cf-4e40-92ec-94aeaca08582.mp4

**Comment by Elaws** [comment]
Thank you, however this work only if the the save can be found in the game folder, which regularly isn't the case. Well, it's not a big deal, just a nice to have option maybe for a future update :)

**Comment by mtkennerly** [comment]
That's true, but that's definitely a tricky case. If the save is stored outside of the game/root folder, there may not be an automated way to tie it to a specific install. It would *probably* be whichever install's executable was run most recently, but I'm not sure if that's tracked, and it might not be true if you opened the other copy and had to quit right away because something came up.

**Comment by Elaws** [comment]
Yeah great remark, I'll have to think about it ;)
